### PR TITLE
Add $MATOMO_DATABASE_TABLE_PREFIX [default: "matomo_"] as wizard option

### PR DESCRIPTION
**Pull request by bebosudo**
### Description of the change

Add an option to set the prefix for database tables.

### Benefits

More options available to adapt this image to existing setups.

### Possible drawbacks

None

### Applicable issues

None

### Additional information

None

### CODE REVIEW

**Comment by carrodher** [comment]
Thanks for creating this PR. Please, note it is needed to sign the commits. Your signature certifies that you wrote the patch or otherwise have the right to contribute the material. You can find more info about this topic, in [the “Sign your work” section from the contributing guidelines](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work).

**Comment by bebosudo** [comment]
Done!
I didn't see tests in the repo, please let me know if I'm missing something.

File: bitnami/matomo/4/debian-11/rootfs/opt/bitnami/scripts/matomo-env.sh [hunk 1]
```diff
@@ -128,6 +129,7 @@ export MATOMO_DATABASE_PORT_NUMBER="${MATOMO_DATABASE_PORT_NUMBER:-3306}" # only
 export MATOMO_DATABASE_NAME="${MATOMO_DATABASE_NAME:-bitnami_matomo}" # only used during the first initialization
 export MATOMO_DATABASE_USER="${MATOMO_DATABASE_USER:-bn_matomo}" # only used during the first initialization
 export MATOMO_DATABASE_PASSWORD="${MATOMO_DATABASE_PASSWORD:-}" # only used during the first initialization
+export MATOMO_DATABASE_TABLE_PREFIX="${MATOMO_DATABASE_TABLE_PREFIX:-}" # only used during the first initialization
```
Comment (marcosbc) [inline bitnami/matomo/4/debian-11/rootfs/opt/bitnami/scripts/matomo-env.sh:132]: For compatibility reasons, we should keep the default `matomo_` value. Could you change it?

```suggestion
export MATOMO_DATABASE_TABLE_PREFIX="${MATOMO_DATABASE_TABLE_PREFIX:-matomo_}" # only used during the first initialization
```

**Review by marcosbc — APPROVED** [review]
Thanks for the contribution! Please have a look at my comment whenever you have a chance.

**Review by marcosbc — CHANGES_REQUESTED** [review]
Sorry, I chose the wrong option. Once it is fixed I will approve it.

File: bitnami/matomo/4/debian-11/rootfs/opt/bitnami/scripts/matomo-env.sh (hunk 1 above)
Comment (bebosudo) [reply bitnami/matomo/4/debian-11/rootfs/opt/bitnami/scripts/matomo-env.sh:132]: Just pushed a fix with fde647c, apologies for the oversight.

Comment (marcosbc) [reply bitnami/matomo/4/debian-11/rootfs/opt/bitnami/scripts/matomo-env.sh:132]: It seems to be the same, could you re-check it?

**Comment by marcosbc** [comment]
The default value for `MATOMO_DATABASE_TABLE_PREFIX` seems to be still empty in `matomo-env.sh`, could you re-check it?

File: bitnami/matomo/4/debian-11/rootfs/opt/bitnami/scripts/matomo-env.sh (hunk 1 above)
Comment (bebosudo) [reply bitnami/matomo/4/debian-11/rootfs/opt/bitnami/scripts/matomo-env.sh]: Sorry about that, forgot to git add the edited file. Should be there now.

**Review by marcosbc — APPROVED** [review]
LGTM, thanks!

**Comment by bebosudo** [comment]
Hi, any update on when would this be available in the [hub](https://hub.docker.com/r/bitnami/matomo/tags)? :-)

**Comment by JoshuaSmeda** [comment]
There's a flaw with this because it doesn't allow for empty ("") values. So if a user doesn't have prefixed tables, they're out of luck because they can't override the behaviour - the ini-file will always default to `matomo_` in this case.

**Pull request merged**